multicluster/cmd/multicluster-controller/options.go

```diff
@@ -51,6 +56,8 @@ func (o *Options) complete(args []string) error {
 			return fmt.Errorf("failed to load options from configuration file %s", o.configFile)
 		}
 		o.options = options
+		o.ServiceCIDR = ctrlConfig.ServiceCIDR
```

Comment:
We should validate it is a valid CIDR string.

Reply:
done